kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -807,20 +807,21 @@ These are just used by the other operators for shuffling local execution state a
            <acctID> ACCT  </acctID>
            <code>   CODE  </code>
            <nonce>  NONCE </nonce>
+           <storage> STORAGE </storage>
            ...
          </account>
-      requires CODE =/=K .Bytes orBool NONCE =/=Int 0
+      requires CODE =/=K .Bytes orBool NONCE =/=Int 0 orBool STORAGE =/=K .Map
 
     rule <k> #newExistingAccount ACCT => .K ... </k>
          <account>
            <acctID>      ACCT      </acctID>
            <code>        CODE      </code>
            <nonce>       0         </nonce>
-           <storage>     _ => .Map </storage>
+           <storage>     STORAGE   </storage>
```

Comment:
Minor optimization. Match here against .Map instead of putting it in the side condition.